src/main/java/org/jongo/RxJongo.java

```diff
@@ -0,0 +1,46 @@
+package org.jongo;
```

Comment:
I think the RxJongo implementation should not be packaged in Jongo. 
The new Function interface allows someone to create its own implementation (reactivex or other). 
We can add a section in the documentation to explain how to add reactiveX with the code you have provided.